pyannote/audio/tasks/segmentation/multilabel.py

```diff
@@ -265,6 +275,26 @@ def validation_step(self, batch, batch_idx: int):
         )
         return {"loss": loss}
 
+    def default_metric(
+        self,
+    ) -> Union[Metric, Sequence[Metric], Dict[str, Metric]]:
+        classes = None
+        if self.classes is not None:
+            classes = self.classes
+        else:
+            classes = self.protocol.stats()["labels"].keys()
```

Comment:
Don't do that. Calling `protocol.stats()` might be veeeeeeeery slow.

Does it ever happen that `classes` is None when `default_metric` is called?
Isn't it always set in `setup(stage="fit")`?

Reply:
Ah ! Indeed it seems that it should never be None.